docs/Quantiles.ipynb

```diff
@@ -0,0 +1,562 @@
+      "execution_count": 4,
+      "metadata": {
+        "id": "3ltU268CXVbM"
+      },
+      "outputs": [],
+      "source": [
+        "context = dp.Context.compositor(\n",
+        "    data=df,\n",
+        "    privacy_unit=dp.unit_of(contributions=1),\n",
+        "    privacy_loss=dp.loss_of(epsilon=1.0),\n",
+        "    split_evenly_over=10,\n",
+        "    margins={\n",
+        "        (\"AGE\", ): dp.Margin(public_info=\"keys\", max_partition_length=60_000_000),\n",
+        "        (): dp.Margin(public_info=\"keys\", max_partition_length=60_000_000),\n",
+        "    },\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XxM09cKUXVbN"
+      },
+      "source": [
+        "## Using the Quantile Method Directly"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Before we get into the quantile method, let's try out the median method directly. We also will need to impute the null values with the mean. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 18,
+      "metadata": {
+        "id": "eryzri-4XVbN"
+      },
+      "outputs": [],
+      "source": [
+        "mean_age = df.select(pl.col(\"AGE\")).mean().collect()\n",
+        "candidates = list(range(100))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Actual Median: 47.0\n",
+            "Differentially Private Median: 46\n"
+          ]
+        }
+      ],
+      "source": [
+        "actual_median = list(df.select(pl.col(\"AGE\").median()).collect())[0][0]\n",
+        "print(f\"Actual Median: {actual_median}\")\n",
+        "\n",
+        "dp_median = list(context.query().select(\n",
+        "    pl.col(\"AGE\").fill_null(mean_age).dp.median(candidates)\n",
+        ").release().collect())[0][0]\n",
+        "print(f\"Differentially Private Median: {dp_median}\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SnUJxheXXVbN"
+      },
+      "source": [
+        "Let's find the age at the 30th quantile! To compute it using the quantile method directly, follow these steps:\n",
+        "\n",
+        "1. Select the variable you are interested in. In this case we chose \"AGE\"\n",
+        "2. Fill in null values. We'll impute the values with the mean age.\n",
+        "3. Specify the following parameters in the quantile method:\n",
+        "\n",
+        "    a. Quantile: This is between 0 and 1. We provide 0.30. \n",
+        "    \n",
+        "    b. A range of possible values for your quantiles. This will depend on your specific domain. We know that ages are generally between 0 and 100, we use those as our candidates, so our results will be limited to these values."
```

Comment:
```suggestion
        "    b. A list of possible values for your quantiles. This will depend on your specific domain. We know that ages are generally between 0 and 100, so we can use those as our candidates, and our results will be limited to these values. If the exact year does not matter to you, you could instead provide multiples of 5 or 10 and consume less of your privacy budget."
```